cms/server/admin/handlers/contest.py

```
@@ -387,25 +362,37 @@ def delete(self, contest_id):
-                assert target_contest_id, (
-                    "Target contest must be specified when moving tasks"
-                )
-                assert target_contest_id != str(contest_id), (
-                    "Target contest cannot be the same as the contest being deleted"
-                )
-
-            self._remove_contest_with_action(contest, action, target_contest_id)
+                if not target_contest_id:
+                    raise ValueError(
+                        "Target contest must be specified when moving tasks"
+                    )
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)